Samples/nuget.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageRestore>
+        <!-- Allow NuGet to download missing packages -->
+        <add key="enabled" value="True" />
+        <!-- Automatically check for missing packages during build in Visual Studio -->
+        <add key="automatic" value="True" />
+    </packageRestore>
+    <packageSources>
+    <clear />
+        <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    </packageSources>
+    <disabledPackageSources />
+    <activePackageSource>
+        <add key="All" value="(Aggregate source)" />
```

Comment:
><add key="All" value="(Aggregate source)" /> [](http://example.com/codeflow?start=8&length=44)

Curious, what does this do? Locally, I've never used it.

Reply:
This is actually not used :) I'll remove
In case you're curious
https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#package-source-sections